src/components/HomePage.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { authService } from '../services/auth.service';
+import {history} from "../helpers/history";
+
+export class HomePage extends Component {
+    constructor(props) {
+        super(props);
+    
+        this.state = {
+          currentUser: null,
+        };
+      }
+
+      componentDidMount() {
+        authService.currentUser.subscribe((x) => {
+          this.setState({
+            currentUser: x,
+          });
+        });
+      }
+
+      logout() {
+        authService.logout();
+        history.push("/login");
+      }
+
+    render() {
+        return (
+            <div className="centeredDiv">
+                
+               <p className="homePageText">HOME PAGE</p>
+               <br/>
+               {this.state.currentUser && (
```

Comment:
@marijana-kolosek Ovde nije nesto lepo formatirano, koristi linter da popravis to malo pre push-a.

Reply:
taj homePage je bio samo privremen da korisnik ima neku stranicu na koju se preusmeri kada se uloguje, sada se preusmerava na stranicu BlogPage, a ovaj HomePage sam izbrisala jer vise nije potreban